browser-extension/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@opentelemetry/browser-extension",
```

Comment:
I moved it into packages and going to name it "opentelemetry-injection-browser-extension", since the purpose is injecting opentelemetry into arbitrary websites.

Reply:
based on @dyladan's comment I renamed it again to opentelemetry-browser-extension-autoinjection